R/get_connection.R

```diff
@@ -172,15 +173,29 @@ get_nuvolos_db_path <- function() {
 #' 
 #' Using outside of Nuvolos only. This helps the user to store credentials safely at local device.
 #' @export
-input_nuvolos_credential <- function(username, password){
+input_nuvolos_credential <- function(){
   # store username & password
+  username <- rstudioapi::askForSecret("Please input your Nuvolos username:")
```

Comment:
Don't these also need updating @IronManJames ?

Reply:
oh, my bad, updated it....